### ISSUE TITLE
1.5.3: sphinx warnings

### ISSUE THREAD

**Issue by kloczek** [getlogbook/logbook#319]
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
/usr/lib/python3.8/site-packages/setuptools/dist.py:697: UserWarning: Usage of dash-separated 'upload-dir' will not be supported in future versions. Please use the underscore name 'upload_dir' instead
  warnings.warn(
running build_sphinx
Running Sphinx v4.1.2
making output directory... done
loading intersphinx inventory from http://docs.python.org/objects.inv...
intersphinx inventory has moved: http://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
building [mo]: all of 0 po files
building [man]: all source files
updating environment: [new config] 24 added, 0 changed, 0 removed
reading sources... [100%] unittesting
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/api/handlers.rst:9: WARNING: duplicate object description of logbook, other instance in api/base, use :noindex: for one of them
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/api/utilities.rst:9: WARNING: duplicate object description of logbook, other instance in api/handlers, use :noindex: for one of them
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-logbook.3 { features quickstart setups stacks performance libraries unittesting ticketing compat api/index api/base api/handlers api/utilities api/queues api/ticketing api/more api/notifiers api/compat api/internal designexplained designdefense cookbook changelog } done
build succeeded, 2 warnings.
```

**Comment by RazerM** [comment]
resolved by #327

**Comment by kloczek** [comment]
Just tested 1.5.3 and I see still some warnings.
In mean time I've switched to use `-n` option
```console
+ /usr/bin/sphinx-build -n -T -b man docs build/sphinx/man
Running Sphinx v6.2.1
making output directory... done
WARNING: The pre-Sphinx 1.0 'intersphinx_mapping' format is deprecated and will be removed in Sphinx 8. Update to the current format as described in the documentation. Hint: "intersphinx_mapping = {'<name>': ('http://docs.python.org', None)}".https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping
loading intersphinx inventory from http://docs.python.org/objects.inv...
intersphinx inventory has moved: http://docs.python.org/objects.inv -> https://docs.python.org/3/objects.inv
building [mo]: targets for 0 po files that are out of date
writing output...
building [man]: all manpages
updating environment: [new config] 24 added, 0 changed, 0 removed
reading sources... [100%] unittesting
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/api/handlers.rst:9: WARNING: duplicate object description of logbook, other instance in api/base, use :noindex: for one of them
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/api/utilities.rst:9: WARNING: duplicate object description of logbook, other instance in api/handlers, use :noindex: for one of them
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-logbook.3 { features quickstart setups stacks performance libraries unittesting ticketing compat api/index api/base api/handlers api/utilities api/queues api/ticketing api/more api/notifiers api/compat api/internal designexplained designdefense cookbook changelog } /home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/quickstart.rst:65: WARNING: py:class reference target not found: logbook.queues.RedisHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/quickstart.rst:183: WARNING: py:attr reference target not found: logbook.Handler.format_string
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/setups.rst:179: WARNING: py:class reference target not found: logbook.queues.RedisHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/libraries.rst:161: WARNING: py:attr reference target not found: logbook.Logger.disabled
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.LoggerMixin.notice:1: WARNING: py:data reference target not found: logbook.NOTICE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.LoggerMixin.trace:1: WARNING: py:data reference target not found: logbook.TRACE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docstring of logbook.Flags.stack_manager:1: WARNING: py:class reference target not found: ContextStackManager
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docstring of logbook.Processor.stack_manager:1: WARNING: py:class reference target not found: ContextStackManager
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docstring of logbook.Handler.stack_manager:1: WARNING: py:class reference target not found: ContextStackManager
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.handlers.TestHandler.has_notice:1: WARNING: py:data reference target not found: NOTICE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.TestHandler.has_notices:1: WARNING: py:data reference target not found: NOTICE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.handlers.TestHandler.has_trace:1: WARNING: py:data reference target not found: TRACE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.TestHandler.has_traces:1: WARNING: py:data reference target not found: TRACE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.handlers.MailHandler.emit_batch:12: WARNING: py:meth reference target not found: handle
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.handlers.MailHandler.get_recipients:1: WARNING: py:attr reference target not found: recipients
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/handlers.py:docstring of logbook.handlers.WrapperHandler:6: WARNING: py:attr reference target not found: _direct_attrs
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.LoggerMixin.notice:1: WARNING: py:data reference target not found: logbook.NOTICE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.set_datetime_format:16: WARNING: py:obj reference target not found: datetime_format
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.set_datetime_format:38: WARNING: py:obj reference target not found: datetime_format
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/queues.py:docstring of logbook.queues.ThreadedWrapperHandler.emit_batch:12: WARNING: py:meth reference target not found: handle
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.TaggingLogger:16: WARNING: py:attr reference target not found: logbook.LogRecord.extra
<unknown>:1: WARNING: py:class reference target not found: logbook.more.TwitterFormatter
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.ExternalApplicationHandler:12: WARNING: py:class reference target not found: logbook.ThreadedWrapperHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.ExceptionHandler.handle:1: WARNING: py:meth reference target not found: emit
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.ExceptionHandler.handle:1: WARNING: py:meth reference target not found: handle_error
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.DedupHandler.handle:1: WARNING: py:meth reference target not found: emit
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/more.py:docstring of logbook.more.DedupHandler.handle:1: WARNING: py:meth reference target not found: handle_error
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/compat.py:docstring of logbook.compat.LoggingHandler.get_logger:1: WARNING: py:attr reference target not found: logger
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.LoggerMixin.notice:1: WARNING: py:data reference target not found: logbook.NOTICE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/logbook/base.py:docstring of logbook.base.LoggerMixin.trace:1: WARNING: py:data reference target not found: logbook.TRACE
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/docs/designexplained.rst:99: WARNING: py:class reference target not found: logbook.more.MultiProcessingHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:246: WARNING: py:class reference target not found: logbook.RotatingFileHandlerBase
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:248: WARNING: py:class reference target not found: logbook.RotatingFileHandlerBase
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:253: WARNING: py:class reference target not found: logbook.more.FingersCrossedHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:279: WARNING: py:class reference target not found: logbook.more.FingersCrossedHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:282: WARNING: py:class reference target not found: logbook.more.FingersCrossedHandler
/home/tkloczko/rpmbuild/BUILD/logbook-1.5.3/CHANGES:308: WARNING: py:class reference target not found: logbook.more.MultiProcessingHandler
done
build succeeded, 40 warnings.
```
You can peak on fixes that kind of issues in other projects
https://github.com/RDFLib/rdflib-sqlalchemy/issues/95
https://github.com/RDFLib/rdflib/pull/2036
https://github.com/click-contrib/sphinx-click/commit/abc31069
https://github.com/frostming/unearth/issues/14
https://github.com/jaraco/cssutils/issues/21
https://github.com/latchset/jwcrypto/pull/289
https://github.com/latchset/jwcrypto/pull/289
https://github.com/pypa/distlib/commit/98b9b89f
https://github.com/pywbem/pywbem/pull/2895
https://github.com/sissaschool/elementpath/commit/bf869d9e
https://github.com/sissaschool/xmlschema/commit/42ea98f2
https://github.com/sqlalchemy/sqlalchemy/commit/5e88e6e8
